wiki/content/get-started/index.md

```diff
@@ -1,115 +1,296 @@
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-2/index.md">}}">
-              2 - Basic Operations
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about UID operations, updating nodes, and traversals.
-          </p>
-        </div>
-      </div>
-      <div class="col-12 col-sm-12">
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-3/index.md">}}">
-              3 - Basic Types and Operations on them
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about data types, indexing, filtering, and reverse traversals.
-          </p>
-        </div>
-      </div>
-      <div class="col-12 col-sm-12">
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-4/index.md">}}">
-              4 - Multi-Language Strings
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about multi-language strings and operations on them using the language tags.
-          </p>
-        </div>
-      </div>
-      <div class="col-12 col-sm-12">
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-5/index.md">}}">
-              5 - String Indices and Modeling Tweet Graph
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about string indices, modeling tweet graph, and keyword-based searching.
-          </p>
-        </div>
-      </div>
-      <div class="col-12 col-sm-12">
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-6/index.md">}}">
-              6 - Advanced Text Search on Social Graphs
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about full-text search and regular expression search.
-          </p>
-        </div>
-      </div>
-      <div class="col-12 col-sm-12">
-        <div class="section-item">
-          <div class="section-name">
-            <a href="{{< relref "tutorial-7/index.md">}}">
-              7 - Fuzzy Search on Social Graphs
-            </a>
-          </div>
-          <p class="section-desc">
-            Learn about searching user names using fuzzy search on social graphs.
-          </p>
-        </div>
-      </div>
-    </div>
-  </div>
-</section>
+{{% notice "note" %}}
+This is a quickstart guide.
+You can find the getting started tutorial series [here]({{< relref "tutorials/index.md" >}}).
+{{% /notice %}}
+
+## Dgraph
+
+Dgraph cluster consists of different nodes (Zero, Alpha & Ratel) and each node serves a different purpose.
+
+**Dgraph Zero** controls the Dgraph cluster, assigns servers to a group,
+and re-balances data between server groups.
+
+**Dgraph Alpha** hosts predicates and indexes.
```

Comment:
This introduces the word predicate (and indexes), with no explanation of what it means.